.github/workflows/gomod-sync.yml

```diff
@@ -0,0 +1,27 @@
+name: go.mod check
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
Do we need this line?

Reply:
Probably not. I copied that part from [exercises-tests.yml](https://github.com/exercism/go/blob/main/.github/workflows/exercise-tests.yml) where we also have that line.

Reply:
Probably from a time when there was a transition between main and master ...

Reply:
Just removed and seems to be working fine.